web/themes/custom/itkdev/itkdev_project_theme/css/project-styles.css

```diff
@@ -4,6 +4,84 @@
     --theme-primary: #fff;
     --theme-secondary: #F4F4F5;
     --theme-brand: #047857;
-    --theme-brand-hover: #064E3B; 
+    --theme-brand-hover: #064E3B;
     --theme-border: #E4E4E7;
-}
\ No newline at end of file
+}
+
+.tablesort {
+  &.desc {
+    height: 100%;
+    vertical-align: text-bottom;
+  }
+
+  &.asc {
+    height: 100%;
+    vertical-align: sub;
+  }
+}
+
+/**
+ * @file
+ *
+ * Replaces standard throbber from stable9 with Olivero's.
+ *
+ * See https://www.drupal.org/docs/develop/theming-drupal/update-the-throbber
+ *
+ * Copied and edited from:
+ * web/core/themes/olivero/css/components/autocomplete-loading.module.css
+ *
+ */
+
+/* remove ugly spinner from autocomplete input  */
+.js input.form-autocomplete {
+  background-image: none;
+}
+
+/* replace spinner, and doing this it only shows when on input and load */
+.js input.form-autocomplete.ui-autocomplete-loading {
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' width='18' height='18' viewBox='0 0 10 10'%3e  %3cstyle type='text/css'%3e%40keyframes s%7b0%25%7btransform:rotate(0deg) translate(-50%25,-50%25)%7d50%25%7btransform:rotate(430deg) translate(-50%25,-50%25);stroke-dashoffset:20%7d100%25%7btransform:rotate(720deg) translate(-50%25,-50%25)%7d%7dellipse%7banimation:s 1s linear infinite%7d%3c/style%3e  %3cg transform='translate(5 5)'%3e    %3cellipse fill='none' ry='4' rx='4' cy='5' cx='5' stroke='%237e96a7' stroke-width='1' stroke-dashoffset='6.125' stroke-dasharray='25' transform='translate(-5 -5)'/%3e  %3c/g%3e%3c/svg%3e");
+  background-position: 98% center;
+}
+
+
+/* Style dropdown from autocomplete input */
+.ui-autocomplete .ui-menu-item {
+  height: 50px;
+  border-bottom: 1px solid rgb(203 213 225); /* bg-slate-300 */
+  color: rgb(71 85 105)
+}
+
+.ui-menu-item:last-child {
+  border-bottom: 0px;
+}
+
+.ui-autocomplete .ui-menu-item-wrapper:focus,
+.ui-autocomplete .ui-menu-item-wrapper:hover {
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.ui-autocomplete .ui-menu-item-wrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.ui-autocomplete .ui-menu-item-wrapper.ui-state-active {
+  margin: 0;
+  border: 0;
+  background-color: rgb(241 245 249); /* bg-slate-100 */
+  color: rgb(71 85 105)
+}
+
+/* This css class is added on the projects view (admin/structure/views/view/projects) */
+.projects-view {
```

Comment:
<img width="1105" alt="image" src="https://github.com/user-attachments/assets/b6b2e658-5d6c-44a0-86cf-4fe5c898b723">

This looks as so, and will change in a later pr